app/styles/base/_typography.scss

```
@@ -0,0 +1,9 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap");
+html {
+  font-size: $font-size-xl;
+  box-sizing: border-box;
+  font-family: "Roboto Slab";
+}
+.activeNav {
```

```suggestion
}

.active-nav {
```
Mỗi block cách nhau ra 1 dòng nhé, ngoài ra tên class để dạng snake như này, CSS đừng để camelCase